core/templates/dev/head/pages/exploration_editor/ExplorationTitleEditor.js

```diff
@@ -0,0 +1,45 @@
+// Copyright 2017 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the exploration title field in forms.
+ */
+oppia.directive('explorationTitleEditor', [
+  'UrlInterpolationService', function(UrlInterpolationService) {
+    return {
+      restrict: 'E',
+      scope: {
+        // The text for the label of the field.
+        labelName: '@',
+        // Value to move focus on the element.
```

Comment:
The focus label used to set focus on the text input field.

(Note that the terminology "label for focus area" is already established in app.js; we can reuse it. https://github.com/oppia/oppia/blob/develop/core/templates/dev/head/app.js#L762 )